tools/configure_server_settings.rb

```diff
@@ -0,0 +1,37 @@
+if ARGV.empty?
+  puts "USAGE: #{__FILE__} server_id settings_path_separated_by_a_/ value [settings_path_separated_by_a_/ value]"
```

Comment:
I don't understand this.  Can we provide an example?

* We should probably require config/environment so it can be run with straight ruby.
`require File.expand_path("../config/environment", __dir__)`

* Can we add a shebang at the top and `chmod +x` these scripts?
`#!/usr/bin/env ruby`